packages/shared/routes/dashboard/wallet/views/Security.svelte

```diff
@@ -144,7 +146,9 @@
             title={locale('views.dashboard.security.version.title', {
                 values: { version: $versionDetails.currentVersion },
             })}
-            message={locale(`views.dashboard.security.version.${$versionDetails.upToDate ? 'upToDate' : 'outOfDate'}`)}
+            message={$stage === Stage.PROD
```

Comment:
We should use get here, might need to import it:
```suggestion
            message={get(stage) === Stage.PROD
```

Reply:
Whoops